plugins/file-open-save-new/api/src/main/java/org/pentaho/di/plugins/fileopensave/api/providers/Entity.java

```diff
@@ -37,4 +39,14 @@ public interface Entity {
   boolean isCanEdit();
   boolean isCanDelete();
   EntityType getEntityType();
+
+  /**
+   * Gets the decoded version of name of an element.
+   * Defaults to getName() {@code String}.
```

Comment:
Should be instead `Defaults to {@link #getName()}.`

Reply:
Addressed.